public/app/features/alerting/unified/components/notification-policies/EditDefaultPolicyForm.tsx

```diff
@@ -59,15 +62,29 @@ export const AmRootRouteForm = ({ actionButtons, alertManagerSourceName, onSubmi
       >
         <div className={styles.container} data-testid="am-receiver-select">
           <Controller
-            render={({ field: { onChange, ref, value, ...field } }) => (
-              <ContactPointSelector
-                selectProps={{
-                  ...field,
-                  onChange: (changeValue) => handleContactPointSelect(changeValue, onChange),
-                }}
-                selectedContactPointName={value}
-              />
-            )}
+            render={({ field: { onChange, ref, value, ...field } }) =>
+              isGrafanaAlertmanager ? (
+                <GrafanaManagedContactPointSelector
+                  onChange={(contactPoint) => {
+                    handleContactPointSelect(contactPoint?.spec.title, onChange);
+                  }}
+                  isClearable
+                  value={value}
+                  placeholder={t(
+                    'alerting.notification-policies-filter.placeholder-search-by-contact-point',
+                    'Choose a contact point'
+                  )}
+                />
+              ) : (
+                <ContactPointSelector
+                  selectProps={{
+                    ...field,
+                    onChange: (changeValue) => handleContactPointSelect(changeValue.value?.name, onChange),
+                  }}
+                  selectedContactPointName={value}
+                />
+              )
```

Comment:
Probably worth moving this into one of the existing components in our core code, as we repeat this pattern?

Reply:
I tried abstracting this into a `HybridContactPointSelector` but the interface to this component is very awkward as many input props would be mutually exclusive :( 

I might take another stab at this in a separate PR.